backend/go.mod

```diff
@@ -167,10 +167,12 @@ require (
 	go.uber.org/multierr v1.11.0 // indirect
 	golang.org/x/crypto v0.14.0 // indirect
 	golang.org/x/mod v0.12.0 // indirect
-	golang.org/x/sys v0.13.0 // indirect
+	golang.org/x/sys v0.15.0 // indirect
 	golang.org/x/term v0.13.0 // indirect
 	golang.org/x/tools v0.13.0 // indirect
 	gopkg.in/linkedin/goavro.v1 v1.0.5 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/testcontainers/testcontainers-go/modules/redpanda => github.com/sago2k8/testcontainers-go/modules/redpanda v0.0.0-20231215004313-2cc387b87cb8
```

Comment:
This is necessary until https://github.com/testcontainers/testcontainers-go/pull/1994 gets merged to upstream.

Reply:
Great PR, hopefully we can have this merged soon, this was missing and should simplify some of the other integration code as well. I should have considered the option to add multiple listeners (or at least one for the docker network) when I wrote the module :(